postmarketOS/pmbootstrap
# Fix #968: cache which packages have been built

Fixes #968.
### How to test

Run this with and without this PR. It should me much faster.
```shell
pmbootstrap build postmarketos-ui-plasma-mobile
```
